src/migrations/20231206210453-update-flags-for-user.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     await queryInterface.sequelize.transaction(async (transaction) => {
       const sessionSig = __filename;
       await prepMigration(queryInterface, transaction, sessionSig);
-      return updateUsersFlagsEnum(queryInterface, transaction, ['training_reports']);
+      // return updateUsersFlagsEnum(queryInterface, transaction, ['training_reports']);
```

Comment:
this shouldn't be in this PR, should it?

Reply:
There were actually two commented out migrations. Not sure why but was having issues updating db.